gatsby-config.js

```diff
@@ -49,6 +49,13 @@ const plugins = [
       path: path.join(__dirname, 'static', 'uploads')
     }
   },
+  {
+    resolve: 'gatsby-source-filesystem',
+    options: {
+      name: 'img',
+      path: path.join(__dirname, 'static')
```

Comment:
Currently `img` contains more than just the documentation images. Should we consider creating a folder for just documentation images, like `static/uploads/images`?

Reply:
Yeah, I think that could help- at least while we discuss whether we want to handle different images with different pipelines instead of through queries at consumption-time. I suspect that isn't possible with our current setup that has docs and blog being completely different MD engines.